src/main/java/technology/tabula/extractors/BatchSelectionExtractor.java

```diff
@@ -0,0 +1,473 @@
+package technology.tabula.extractors;
+
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.util.ArrayList;
+import java.util.List;
+
+import org.apache.commons.cli.ParseException;
+import org.apache.pdfbox.pdmodel.PDDocument;
+
+import technology.tabula.ObjectExtractor;
+import technology.tabula.Page;
+import technology.tabula.PageIterator;
+import technology.tabula.Rectangle;
+import technology.tabula.Table;
+import technology.tabula.detectors.NurminenDetectionAlgorithm;
+import technology.tabula.detectors.StringSearch;
+import technology.tabula.detectors.SpreadsheetDetectionAlgorithm;
+import technology.tabula.writers.CSVWriter;
+import technology.tabula.writers.Writer;
+
+// NOTES:
+//		need to remove tables from auto/spread list if they are used as a best guess
+//		or remove very similar tables from the list before extracting data
+public class BatchSelectionExtractor {
+
+	public int extract(String inputPath, String outputPath, String jsonPath, String processType, boolean ocrAllowed,
+			int overlapThreshold) {
+		System.out.println("OCR Selection: " + ocrAllowed);

```

Comment:
Remove print to `stdout`. If needed, use a Logger. See example here: https://github.com/tabulapdf/tabula-java/blob/master/src/main/java/technology/tabula/ObjectExtractorStreamEngine.java#L96